docs/product-indexing.md

```diff
@@ -0,0 +1,31 @@
+# Product Indexing
+
+Product indexing is used to improve the search performance and speed.
+
+The Index widget, located on Product details screen, compares the date of product indexing in the search engine with the current indexing date.
+
+VirtoCommerce reindexes automatically whenever one or more items are changed (for example, price changes, catalog or shopping cart price rules are created, new categories added, and so on). Reindexing is performed as a background process. However, the indexing can be done also manually using the 'Build index' functionality.
+
+## Indexing Widget
+
+### Access Indexing Widget
+
+1. In order to access the Indexing widget, the user should open the Product details screen; 
+1. The Indexing widget displays the information about when the product was last indexed;
+1. The user clicks on the Indexing widget;
+1. The system will open the 'Search index details' of the product and the Index content (code) will be displayed. 
```

Comment:
"Index content (code)" it is not a code, it is index document content. If you want to specify something, it could be "Index content (json)"